### PR TITLE
[DOC] add interactive bundle viz to OR example

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

**Comment by arokem** [comment]
A couple of thoughts here: 

1. For some reason, the plotly plot appears at the very top of this example, unlike the other examples where it appears in the end. I couldn't really find the code reason for it, though. 
2. Should we do only the OR in this example? I think that the visualization will be a bit clearer in that case. 
3. I see that the code also generates a montage. Maybe we should pull in the montage to display, instead of the standard html visualization? It's not interactive, but it would at least demonstrate this functionality. We could also do both.

Also, noticed that the ORs in this subject also have the additional more anterior section:

Screen Shot 2022-07-20 at 9 27 10 AM

**Comment by 36000** [comment]
I think putting the montage only is a good idea. The cropped endpoint ROIs would fix the anterior section problem

**Review by arokem — COMMENTED** [review]
A couple of comments here. 

I didn't realize that this also involves a new set of OR ROIs! We should ask @SendyCaffarra and @jyeatman for some feedback on this new definition of the anterior portion of the OR, which should help get rid of the anterior branch that we have been seeing in some subjects.

File: AFQ/data/fetch.py [hunk 1]
```diff
@@ -593,9 +593,9 @@ def read_templates(as_img=True, resample_to=False):
 ]
 
 or_remote_fnames = [
-    "26831630",
+    "36514170",
```
[the remark this answers is not in the document]
Comment (36000) [reply AFQ/data/fetch.py:591]: Yes and this works well in test datasets

**Comment by 36000** [comment]
@arokem  I am not sure if the sphinx integration is good to go, but otherwise this is good. for example the new thalamus ROI is tested on hbn, hcp, and hardi, and it gets rid of the anterior endpoint problem.

File: AFQ/data/fetch.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (arokem) [reply AFQ/data/fetch.py:591]: Do we have a way to refer to different versions of this data on figshare?

**Comment by arokem** [comment]
@36000 : when you get a chance, could you please rebase this on top of current master?

File: AFQ/data/fetch.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (36000) [reply AFQ/data/fetch.py:591]: I am not sure what you mean by this. Note that the new OR ROIs are on my figshare but the old ones are not.

[the remark this answers is not in the document]
Comment (arokem) [reply AFQ/data/fetch.py:591]: Oh! Gotcha. I think that means we are good to go here.

[the remark this answers is not in the document]
Comment (arokem) [reply AFQ/data/fetch.py:591]: I mean up to the sphinx stuff, which I can sort out. Let me pull this rebased thing down and fiddle with that.

[the remark this answers is not in the document]
Comment (arokem) [reply AFQ/data/fetch.py:591]: Oh - I see you are sorting that out too in https://github.com/yeatmanlab/pyAFQ/pull/861/commits/a8a12a14469cd9f6e0170d3bfbe0d96339a7f327

[the remark this answers is not in the document]
Comment (36000) [reply AFQ/data/fetch.py:591]: I still need to test that

**Comment by arokem** [comment]
Looks like it works now!

**Pull request merged**